xml/hwbp_registers.xml

```diff
@@ -820,17 +869,27 @@
         tie any number of upper bits to 0. The $|select|$ bits may only support
         0 (ignore).
 
-        <field name="mvalue" bits="31:26" access="WARL" reset="0">
-            Data used together with \FcsrTextraThirtytwoMselect.
+        <field name="mhvalue" bits="31:26" access="WARL" reset="0">
+            Data used together with \FcsrTextraThirtytwoMhselect.
         </field>
-        <field name="mselect" bits="25" access="WARL" reset="0">
-            0: Ignore \FcsrTextraThirtytwoMvalue.
+        <field name="mhselect" bits="25:23" access="WARL" reset="0">
+            000: Ignore \FcsrTextraThirtytwoMhvalue.
 
-            1: This trigger will only match if the low bits of
-            \RcsrMcontext equal \FcsrTextraThirtytwoMvalue.
+            100: This trigger will only match if the low bits of
+            \RcsrMcontext/\RcsrHcontext equal \FcsrTextraThirtytwoMhvalue.
+
+            x01: This trigger will only match if the low bits of
+            \RcsrMcontext/\RcsrHcontext equal {\FcsrTextraThirtytwoMhvalue, mhselect[2]}.
+
+            x10: This trigger will only match if VMID in satp equals the lower VMIDMAX
+            (defined in the Privileged Spec) bits of {\FcsrTextraThirtytwoMhvalue, mhselect[2]}.
```

Comment:
The braces in these two descriptions are not being rendered in the PDF.

Reply:
I missed that.  I'll push the fix in a minute.